proto/tendermint/budget/v1beta1/query.proto

```diff
@@ -0,0 +1,127 @@
+syntax = "proto3";
+
+package cosmos.budget.v1beta1;
+
+import "tendermint/budget/v1beta1/budget.proto";
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+import "google/api/annotations.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "cosmos_proto/cosmos.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+option go_package = "github.com/crescent-network/crescent/v5/x/budget/types";
+
+// Query defines the gRPC query service for the budget module.
+service Query {
+  // Params returns parameters of the budget module.
+  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
+    option (google.api.http).get                                           = "/cosmos/budget/v1beta1/params";
```

Comment:
I think it'd be good to change this URL to `/crescent/budget/v1beta1/params`, but it can break other existing clients that rely on this query.

Reply:
ditto, This PR should never include state, api breaking changes